Makefile

```
@@ -172,6 +175,28 @@ e2e_test: install
 		$(NULL)
 
+# e2e-hyperfleet: runs the hyperfleet Platform API sanity test.
+#
+# Required:
+#   HYPERFLEET_URL  — Platform API v2 base URL
+#
+# Optional:
+#   CLUSTER_NAME       — defaults to hf-e2e-<unix timestamp> (≤18 chars)
+#   AWS_DEFAULT_REGION — fallback when region cannot be derived from HYPERFLEET_URL
+.PHONY: e2e-hyperfleet
+e2e-hyperfleet: install
```

checkmake (0.3.2)

[warning] Target body for "e2e-hyperfleet" exceeds allowed length of 5 lines (11).

(maxbodylength)